packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -102,6 +106,19 @@ export function BlockSettingsDropdown( {
 	const { getBlockOrder, getSelectedBlockClientIds } =
 		useSelect( blockEditorStore );
 
+	const { openedBlockSettingsMenu } = useSelect( ( select ) => {
+		const { getOpenedBlockSettingsMenu } = unlock(
+			select( blockEditorStore )
+		);
+		return {
+			openedBlockSettingsMenu: getOpenedBlockSettingsMenu(),
+		};
+	}, [] );
```

Comment:
```suggestion
	const openedBlockSettingsMenu = useSelect(
		( select ) => unlock( select( blockEditorStore ) ).getOpenedBlockSettingsMenu(),
		[]
	);
```


Completely optional and utility-free optimization 😄

Reply:
Good idea! I usually draft it up with the longer method to make it easier to add other properties in there while hacking around, then always forget to simplify it again 😄

Committed in https://github.com/WordPress/gutenberg/pull/54083/commits/c59e97405aaecdaa7ca1a195c63bfeca1c776a8f to please the linter.